types/selenium-webdriver/remote.d.ts

```diff
@@ -158,6 +158,53 @@ export namespace DriverService {
 }
 
 /**
+ * Manages the life and death of the
+ * <a href="http://selenium-release.storage.googleapis.com/index.html">
+ * standalone Selenium server</a>.
+ */
+export class SeleniumServer extends DriverService {
+    /**
+     * @param {string} jar Path to the Selenium server jar.
+     * @param {SeleniumServer.Options=} opt_options Configuration options for the
+     *     server.
+     * @throws {Error} If the path to the Selenium jar is not specified or if an
+     *     invalid port is specified.
+     **/
+    constructor(jar: String, opt_options: SeleniumServer.Options);
+}
+
+export namespace SeleniumServer {
+    /**
+     * Options for the Selenium server:
+     *
+     * - `loopback` - Whether the server should only be accessed on this host's
+     *     loopback address.
+     * - `port` - The port to start the server on (must be > 0). If the port is
+     *     provided as a promise, the service will wait for the promise to resolve
+     *     before starting.
+     * - `args` - The arguments to pass to the service. If a promise is provided,
+     *     the service will wait for it to resolve before starting.
+     * - `jvmArgs` - The arguments to pass to the JVM. If a promise is provided,
+     *     the service will wait for it to resolve before starting.
+     * - `env` - The environment variables that should be visible to the server
+     *     process. Defaults to inheriting the current process's environment.
+     * - `stdio` - IO configuration for the spawned server process. For more
+     *     information, refer to the documentation of `child_process.spawn`.
+     *
+     * @typedef {{
```

Comment:
The comment shouldn't be defining types, the actual code should.

Reply:
From the name `Options`, I imagine all of these fields are optional?

```
Options {
  loopback?: boolean|undefined,
  ...
}
```